assets/src/scss/_media-queries.scss

```diff
@@ -17,17 +17,19 @@
 	@include demo-site-templates--laptop();
 }
 
-
 @media (min-width: #{$desktop}) {
 	@include general--desktop();
 	@include ob-editor-tabs--desktop();
 	@include library--desktop();
 	@include demo-site-templates--desktop();
 }
 
-
 @media (min-width: #{$desktop-large}) {
 	@include general--desktop-large();
 	@include card--desktop-large();
 	@include ob-editor-tabs--desktop-large();
 }
+
+@media( max-width: #{$tablet}){
```

Comment:
@UVLabs The CSS for this is mobile-first and only uses min-width media queries. Let's respect the convention and to the styles like that 👍🏻